crates/stages/src/stages/total_difficulty.rs

```diff
@@ -69,6 +72,30 @@ impl<DB: Database> Stage<DB> for TotalDifficultyStage {
         for entry in walker {
             let (key, header) = entry?;
             td += header.difficulty;
+
+            if self.chain_spec.fork(Hardfork::Paris).active_at_ttd(td) {
```

Comment:
Ah I see, this is the check which we said we'd have to pull out to the TD stage. Makes sense. I was hoping we could maybe skip running this stage for blocks after the merge, but makes sense to do this so that we run the validation rules. It does mean that we do validation in 2 different places (one in consensus, one here).